client/my-sites/hosting/main.js

```diff
@@ -1,40 +1,131 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Main from 'components/main';
-import Banner from 'components/banner';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
 import canSiteViewAtomicHosting from 'state/selectors/can-site-view-atomic-hosting';
 import SFTPCard from './sftp-card';
 import PhpMyAdminCard from './phpmyadmin-card';
 import SupportCard from './support-card';
 import PhpVersionCard from './php-version-card';
 import { isEnabled } from 'config';
+import NoticeAction from 'components/notice/notice-action';
+import TrackComponentView from 'lib/analytics/track-component-view';
+import Notice from 'components/notice';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import isAutomatedTransferActive from 'state/selectors/is-automated-transfer-active';
+import { transferStates } from 'state/automated-transfer/constants';
+import { requestSite } from 'state/sites/actions';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const Hosting = ( { translate, isDisabled, canViewAtomicHosting, siteId } ) => {
+const Hosting = ( {
+	canViewAtomicHosting,
+	clickActivate,
+	isDisabled,
+	isTransferring,
+	requestSiteById,
+	siteId,
+	siteSlug,
+	translate,
+	transferState,
+} ) => {
 	if ( ! canViewAtomicHosting ) {
 		return null;
 	}
 
-	const sftpPhpMyAdminFeaturesEnabled =
-		isEnabled( 'hosting/sftp-phpmyadmin' ) && siteId > 168768859;
+	const getAtomicActivationNotice = () => {
+		const { COMPLETE, FAILURE } = transferStates;
+
+		// Transfer in progress
+		if (
+			( isTransferring && COMPLETE !== transferState ) ||
+			( isDisabled && COMPLETE === transferState )
+		) {
+			requestSiteById( siteId );
+
+			return (
+				<Fragment>
```

Comment:
Minor: we can use the [fragment shorthands](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html) wich removes the neccesity of importing `Fragment`:

```suggestion
				<>
```